lib/GeoIP2/Record/Location.pm

```diff
@@ -78,6 +90,20 @@ IP address is likely to be.
 
 This attribute is only available from the Insights end point.
 
+=head2 $location_rec->average_income()
+
+This returns a non-negative integer representing the average income in US
+dollars associated with the requested IP address.
+
+This attribute is only available from the Insights end point.
+
+=head2 $location_rec->estimated_population()
+
+Returns a non-negative integer representing the estimated population per square
+mile associated with the requested IP address.
```

Comment:
Good catch. That's documentation left over from before we converted to using kilometres